go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/vmware-tanzu/velero
 
-go 1.23
+go 1.23.0
```

Comment:
Do we need to bring back the .0 here, or is 1.23 fine here?

Reply:
We need .0 because https://go.googlesource.com/crypto/+/refs/tags/v0.35.0/go.mod#3 introduced it.

Reply:
unfortunately